react/package.json

```diff
@@ -23,6 +23,7 @@
     "lint-fix": "eslint --fix . && exit 0",
     "prepublishOnly": "npm run build",
     "storybook": "start-storybook -p 6006 -s ./node_modules/@massds/mayflower-assets",
+    "storybook:extract": "sb extract",
```

Comment:
This command is provided by the `@storybook/cli` package.

Reply:
This command extracts information about our storybook instance out into a `.json` data file that it places into the storybook-static/ directory. We can then use this information about all of our stories for whatever we want - in this case I use it for looping over the list of stories for backstop.